src/js/profile/mobile/widget/Calendar.js

```diff
@@ -110,22 +113,30 @@
 						day = day - 1;
 						leftDays = leftDays - 1;
 						cell = row.insertCell();
-						cell.innerHTML = prevLastDate - day;
-						cell.style.opacity = "10%";
+						cell.insertAdjacentHTML("afterbegin", divElement);
+						cell.firstChild.innerHTML = prevLastDate - day;
+						cell.firstChild.style.opacity = "10%";
 					}
 					while (leftDays != 0) {
 						if (setDays > prevLastDate) { // next disabled days.
 							cell = row.insertCell();
-							cell.innerHTML = nextMonthDate;
-							cell.style.opacity = "40%";
+							cell.insertAdjacentHTML("afterbegin", divElement);
+							cell.firstChild.innerHTML = nextMonthDate;
```

Comment:
you many times use `cell.firstChild`, so you can assign it to local variable

Reply:
```
cell.insertAdjacentHTML("afterbegin", divElement);
div = cell.firstChild;
div.innerHTML = = nextMonthDate;
```

Reply:
done.

Reply:
this is small fix but make difference in bigger view. When the file is changed to tau.min.js these properties can be smaller

Reply:
I agree. It seems that clear.